package.json

```diff
@@ -177,5 +177,9 @@
       "waitsForPromise",
       "indexedDB"
     ]
+  },
+  "devDependencies": {
+    "chai": "^3.5.0",
+    "mocha": "^2.5.1"
```

Comment:
Good question! 👍 

I think Atom needs those dependencies at runtime, so we should just probably put them under the root's `package.json` as we do for jasmine. On the other hand, package authors won't be able to use it anyways, so it's not really useful to include it in the final executable (although it seems like even when in `devDependencies`, those packages are included in the final asar archive 😕).

I'll put those two dependencies under the regular dependencies (to be consistent with jasmine) and we can refine this later if needed.